content/en/docs/concepts/security/multi-tenancy.md

```diff
@@ -86,7 +86,7 @@ methods comes with its own set of tradeoffs that impact the isolation level, imp
 effort, operational complexity, and cost of service.
 
 A Kubernetes cluster consists of a control plane which runs Kubernetes software, and a data plane
-consisting of worker nodes where tenant workloads are executed as pods. Tenant isolation can be
+consisting of nodes where tenant workloads are executed as pods. Tenant isolation can be
```

Comment:
the authors here really meant "worker nodes run workloads", i don't see what's wrong with that.